#request
RecordingConfig config
---
#response
RecordingConfig config